### PR TITLE
Build Hello- Microverse

### PR DESCRIPTION
**Pull request by AceMannie**
This Pull request ,  I have added the requirements. 

- [x] Add `index.html` file with a header.
- [x] Add `style.css` with a style for the header.  
- [x]  Add `text.md` file that should be ignored by git.
- [x] Created `.gitignore` file and add `text.md `and `node_modules` to it.

Note: This pull request is continuation of previous [pull request](https://github.com/AceMannie/Hello-Microverse/pull/1).

The previous reviewer asked me to create a new branch with a descriptive name, which i have done here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,63 @@
...
+**Server**
+Express.js
+**Database**
+PostgreSQL
+### Key Features
+- **[When clicked it prints "Hello Microverse" in aquamarine color]**
+## :rocket: Live Demo
+- [No Live Demo Link]
+## :computer: Getting Started
+prints Hello Microverse in default browser.
```
Comment (DuaneDave) [inline README.md]: As previously requested, the getting started section and its subsection are supposed to have details that will instruct new users of your projects on what they need and need to know before using your app. Kindly fill in the correct details for the sections to make your documentation more effective.

You can use this [one](https://github.com/Christelle-12/Mobile-project) for reference

File: README.md [hunk 2]
```diff
@@ -1 +1,63 @@
...
+### Prerequisites
+Clone this repository to your desired folder and open html file.
```
Comment (DuaneDave) [inline README.md]: The prerequisites section should contain the things a new user should know before being able to use the project. I recommend you include this section correctly to make your readme more effective in instructing a new user using the above-linked readme as a reference.

File: README.md [hunk 3]
```diff
@@ -1 +1,63 @@
...
+MIT License
+A short and simple permissive license with conditions only requiring preservation of copyright and license notices. Licensed works, modifications, and larger works may be distributed under different terms and without source code.
+Dec 28th, 2022
+GitHubGitHub
+Emmanuel - Overview
+Emmanuel has 12 repositories available. Follow their code on GitHub.
+Choose a LicenseChoose a License
+MIT License
+A short and simple permissive license with conditions only requiring preservation of copyright and license notices. Licensed works, modifications, and larger works may be distributed under different terms and without source code.
+Dec 28th, 2022
```
Comment (DuaneDave) [inline README.md]: [OPTIONAL] This section is not supposed to be found in the readme but should be the content of a file called LICENSE in your project filesystem. Kindly remove it to declutter the documentation.

File: README.md [hunk 4]
```diff
@@ -1 +1,63 @@
...
+- [ ] **[Add some Javascript]**
+## :handshake: Contributing
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](../../issues/).
+## :star:️ Show your support
+If you like this project feel free to share with your friends.
+## :pray: Acknowledgments
+I would like to thank Microverse for this amazing exercise.
+## :memo: License
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed.
```
Comment (DuaneDave) [inline README.md]: There should be a LICENSE file in your project filesystem and this file will contain what you have added below this line. After including that file, the link to the file is not  `https://choosealicense.com/licenses/mit/` but ./LICENSE. Consider implementing this in order to make the readme more personalized.

**Review by DuaneDave — CHANGES_REQUESTED** [review]
Hi @AceMannie 👋 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.


 ## Highlights
✔️ Good commit history
✔️ Great PR summary
✔️ Correct usage of GitHub flow


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (AceMannie) [reply README.md]: @DuaneDave  kindly  review the  updated Hello Microverse Project README.md file. 

Thank You.

**Review by RaoAkif — APPROVED** [review]
## STATUS: APPROVED 🎉🎉🎉

Hi @AceMannie, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

approve-image

## Highlights
✔ Linters are on point 
✔ Splendid work with the README file
✔ Great use of GitHub flow
✔ Good code design and implementation

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ibizugbe — APPROVED** [review]
STATUS: APPROVED 🎉🎉🎉

Hi @AceMannie,

Your project is complete! There is nothing else to say other than... it's time to merge it ☺️
Congratulations! 🎉


Highlights☺️☺️

✔ Linters are passing
✔ Readme file looks good.
✔ Great use of GitHub flow

Optional suggestions
_no optional suggestions_

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

As described in the Code reviews limits policy you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this form.

**Comment by AceMannie** [comment]
Thank you for reviewing the code @Ibizugbe  to understand you more on the [OPTIONAL] prefix, are you talking about the README.md file in  line 22 -  26? please I'd like some clarifications .

Thank you.

**Comment by Ibizugbe** [comment]
Hello  @AceMannie , I don't think I dropped any optional suggestion, probably you can point out what you are referring to if there is any form of misunderstanding😉😉

**Comment by AceMannie** [comment]
@Ibizugbe  Thank you for your response, I understand the project has been approved .

However in the dashboard I'm still seeing a submit button appeared instead of the green complete icon so i thought there's something needed to be done with your comment below .

"Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application."   Can i go ahead and click for submit and merge the pull request?

**Comment by Ibizugbe** [comment]
@AceMannie , I understand you clearly. When your project is completed, a form will be provided for you to submit the link to your project. When this is submitted, your project is marked completed☺️ with the green badge😉

You can also see before the optional those texts that it was written "no optional suggestions" those texts are there to just let you know more about optional suggestions.. you can kindly ignore it for now.. and take note of it only when you have an "optional suggestion"

Incase you're still confused,  Kindly send a message on slack, then we can proceed to a zoom call if needed.

**Comment by AceMannie** [comment]
Thank you for the clarity @Ibizugbe  have check the form and submitted . will now close the comment.

**Pull request merged**